Changes.md

```diff
@@ -1,6 +1,11 @@
 1.x.x.x
 =======
 
+Features
+--------
```

Comment:
This PR needs to be for `1.0_maintenance`, with the changelog entry going into the `1.0.x` section.